models/token.js

```diff
@@ -17,7 +17,7 @@ const MODEL = {
 
     hash: Joi
         .string()
-        .base64()
+        .regex(/[a-zA-Z0-9_-]+~/)
```

Comment:
We could, if there's some reason those are preferable. There are only three characters we need to replace though.